docs/content/_index.md

```diff
@@ -47,15 +75,24 @@ The results of running Monkey Agents are:
    discovered and/or exploited.
 
 A more in-depth description of reports generated can be found in the [reports
-documentation page]({{< ref "/reports" >}}).
+documentation page.]({{< ref "/reports" >}})
+
+### Infection Map
+![Infection Map](/images/island/infection_map_page/infection_map.png "Infection Map")
+
+### Security Report
+![Security Report](/images/island/reports_page/security_report_overview.png "Security Report")
 
 ## Getting Started
 
-If you haven't downloaded Infection Monkey yet you can do so [from our
-download page](https://github.com/guardicore/monkey/releases/latest). After
-downloading the Monkey, install it using one of our [setup guides]({{< ref
-"/setup" >}}), and read our [getting started guide]({{< ref
-"/usage/getting-started" >}}) for a quick-start on Monkey!
+If you're completely new to Infection Monkey, you can work through the
+[tutorials](/tutorials/) to get your feet wet.
+
+If you have a basic understanding of Infection Monkey, you can download it
```

Comment:
This makes it seem like I should do he tutorials before I download Infection Monkey... but don't I need Infection Monkey to do the tutorials? 😵‍💫

Reply:
You don't, though :)

Reply:
Are you saying that since we have an "easy button" for the tutorials, that you're differentiating it from a typical Infection Monkey installation?

Reply:
Yep. The installation we use for the tutorials isn't an official release, at least not at the moment. Perhaps we should mention that in Tutorial 0.

Reply:
![image](https://github.com/guardicore/monkey/assets/19957806/de4b335f-e63a-4efc-b11a-7b9b02c88383)